src/Sulu/Component/Content/Tests/Unit/Metadata/Loader/StructureXmlLoaderTest.php

```diff
@@ -294,22 +293,6 @@ public function testLoadRequiredTagOtherType()
         $this->assertCount(2, $properties);
     }
 
-    public function testLoadFormWithBlockTypeProperty()
-    {
-        $this->expectException(ReservedPropertyNameException::class);
-        $this->expectExceptionMessageRegExp('"type"');
-
-        $this->load('template_with_block_type_property.xml');
-    }
-
-    public function testLoadFormWithBlockSettingsProperty()
-    {
-        $this->expectException(ReservedPropertyNameException::class);
-        $this->expectExceptionMessageRegExp('"settings"');
-
-        $this->load('template_with_block_settings_property.xml');
-    }
-
```

Comment:
Why have you removed these tests?

Reply:
They don't work anymore, because this is a unit test and the `StructureFormMetadataLoader` (which is calling the validator) never gets called.